styleguide.config.js

```diff
@@ -10,6 +10,10 @@ module.exports = {
   title: "nteract components",
   defaultExample: false,
   sections: [
+    {
+      name: "Data Explorer",
+      components: "packages/transform-dataresouce/src/*.js"
```

Comment:
looks like a little typo

```suggestion
      components: "packages/transform-dataresource/src/*.js"
```

Reply:
:facepalm: